app/utils/shared/customValidate.ts

```diff
@@ -4,14 +4,15 @@ export const isValidKeycloakURI = (uri: string) => {
     new URL(uri);
     if (uri !== uri.trim()) return false;
     if (uri.match(/\s/)) return false;
-    if (!uri.match(/^https?:\/\//)) return false;
+    if (!uri.match(/^[a-zA-Z]+?:\/\//)) return false;
```

Comment:
Thank you for pointing this out!
I believe `?` would be an extra in this case?

Reply:
> Hello @micheal-w-wells, thank you for the contribution! I just realized our CI actions didn't run on outside forks since they triggered on push, I will change that. Would you mind checking that the front-end tests are passing? The test suite `app/jest/helpers.test.ts` should be failing, for the test:
> 
> ```js
> describe('kecloak URIs', () => {
>   it('should validate the URI correctly', () => {
>     expect(isValidKeycloakURI('http://a')).toBe(true);
>     expect(isValidKeycloakURI('https://a')).toBe(true);
>     expect(isValidKeycloakURI('shttps://a')).toBe(false);
>     expect(isValidKeycloakURI('htttps://a')).toBe(false);
>     expect(isValidKeycloakURI('https://')).toBe(false);
>     expect(isValidKeycloakURI('http://')).toBe(false);
>     expect(isValidKeycloakURI('http:/')).toBe(false);
>     expect(isValidKeycloakURI(' http:/')).toBe(false);
>     expect(isValidKeycloakURI('http://a ')).toBe(false);
>     expect(isValidKeycloakURI('http://a b')).toBe(false);
>   });
> });
> ```
> 
> the third and fourth exception should fail but can be removed to support custom URIs

will do!

Reply:
No worries!  Yes the `?` would mean that `://` would be valid which I think is a bug